contracts/Registry.vy

```diff
@@ -881,6 +1032,7 @@ def remove_pool(_pool: address):
     # remove _pool from pool_list
     location: uint256 = self.pool_data[_pool].location
     length: uint256 = self.pool_count - 1
+    is_meta: bool = self.pool_data[_pool].base_pool != ZERO_ADDRESS
```

Comment:
`is_meta` isn't required until the loop starting at line 1070 - declaring it so early on means we're allocating more memory than we have to and makes all the private calls more expensive. I'd declare it at line 1069 instead.